halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/openstack/OpenstackAccount.java

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.halyard.config.model.v1.providers.openstack;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
+import com.netflix.spinnaker.halyard.config.model.v1.node.LocalFile;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
+import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class OpenstackAccount extends Account {
+    private String authUrl;
+    private String username;
+    private String password;
+    private String projectName;
+    private String domainName;
+    private Boolean insecure = false;
+    @LocalFile private String userDataFile;
+    private OpenstackLbaasOptions lbaas = new OpenstackLbaasOptions();
+    private String regions;
+
+    //TODO(emjburns): add support for rosco options
```

Comment:
I just mean the specific openstack rosco options, detailed at  http://www.spinnaker.io/v1.0/docs/target-deployment-configuration#section-openstack under "Rosco configuration". There are extra openstack options, which is why I put the todo here. Is that logic ok, or should I move it to `OpenstackProvider`?